docs/dev/adding_instruments.rst

```diff
@@ -236,10 +236,10 @@ The simplest version, suitable when the instrument connection needs default sett
 
 .. code-block:: python
 
-    def __init__(self, adapter, **kwargs):
+    def __init__(self, adapter, name="Extreme 5000", **kwargs):
```

Comment:
Minor comment on line 214 above: a blank line needs to be added as follow:
```
The status property could look like this:
<blank line here>
.. testcode::

```

Reply:
Done.